### ISSUE TITLE
Elevated circulating beta chorionic gonadotropin concentration

### ISSUE THREAD

**Issue by MickeySegal** [obophenotype/human-phenotype-ontology#10862]
**Preferred term label:**
Elevated circulating beta chorionic gonadotropin concentration

**Synonyms**
HCG

**Definition (free text, please give PubMed ID)**

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
Germ cell tumors, central nervous system

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
@MickeySegal 

See [Elevated circulating beta chorionic gonadotropin concentration HP:6000485](https://hpo.jax.org/browse/term/HP:6000485)

**Comment by MickeySegal** [comment]
Sorry, this was meant to be:
Elevated beta chorionic gonadotropin concentration in **CSF**

**Comment by pnrobinson** [comment]
@MickeySegal 
Adding new term: 
- ID: HP:6001299
- label: Elevated CSF beta chorionic gonadotropin concentration
- parents: [Abnormal CSF protein concentration (HP:0025456)](https://hpo.jax.org/app/browse/term/HP:0025456)
- pmid: PMID:11007160
- def: The concentration of beta chorionic gonadotropin in the cerebrospinal fluid (CSF) is above the upper limit of normal.
- comment: CSF is marker for monitoring tumor recurrence or evaluation of treatment results in individuals with malignant germ cell tumor.
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/10862
- synonyms:
